### PR TITLE
Assinment

### PR DESCRIPTION
**Pull request by mehdihasanpour**
Hi @mohammadhsn 
resolve of oop practices.

### CODE REVIEW

File: php/oop/assignments/zoo.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    {
+        return 'I can move like ' . $this->name;
+    }
+}
+
+class WildAnimal extends Animal
+{
+    public function hunt(Animal $animal)
+    {
+        if (is_subclass_of($animal, 'WildAnimal')) {
```
Comment (mohammadhsn) [inline php/oop/assignments/zoo.php]: I suggest `WildAnimal::class` instead of `WildAnimal`

**Review by mohammadhsn — CHANGES_REQUESTED** [review]
There is no method for parturition

File: php/oop/assignments/zoo.php (hunk 1 above)
Comment (mehdihasanpour) [reply php/oop/assignments/zoo.php]: Hi 
I change the requested part and also add parturition method by  using trait but i don't know this method is correct or not.

**Comment by mehdihasanpour** [comment]
Hi @mohammadhsn 
I change the requested part and also add parturition method by using trait but i don't know this approach is correct or not.

**Comment by mohammadhsn** [comment]
It's better now, but the issue is where all kinds of animals have the `parturition()` method

**Pull request closed without merging**